main.js

```diff
@@ -11,6 +11,20 @@
         maxBoundsViscosity: 0.90
     });
 
+    var sidebar = document.getElementById("sidebar");
+    var sidebarToggle = document.getElementById("sidebarToggle");
+
+    //Start with sidebar closed if mobile
+    if (screen.width <= 800) {
+        sidebarToggle.open = 0;
+        sidebar.style.display = "none";
```

Comment:
Isn't this the problem we were having with Safari? What if we created `.hidden` and `.shown` classes that had the appropriate `display` attribute? And we could assign and remove that